components/mediation/mediators/oauth-mediator/org.wso2.micro.integrator.mediator.oauth/src/main/java/org/wso2/micro/integrator/mediator/oauth/config/xml/OAuthMediatorFactory.java

```diff
@@ -0,0 +1,91 @@
+/*
+ *  Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.micro.integrator.mediator.oauth.config.xml;
+
+
+import org.apache.axiom.om.OMAttribute;
+import org.apache.axiom.om.OMElement;
+import org.apache.synapse.Mediator;
+import org.apache.synapse.config.xml.AbstractMediatorFactory;
+import org.apache.synapse.config.xml.XMLConfigConstants;
+import org.wso2.micro.integrator.mediator.oauth.OAuthMediator;
+import org.wso2.micro.integrator.mediator.oauth.MediatorException;
+
+import javax.xml.namespace.QName;
+import java.util.Properties;
+
+/**
+ * <oauthService remoteServiceUrl = "https://identityserever/services/OAuthService"
+ * />

```

Comment:
Shall we merge this to the previous line?

Reply:
Done in 563f10f6e2d2e0936ec3843f9ae35d84a369bef9